rdrf/rdrf/static/js/show_password.js

```diff
@@ -0,0 +1,50 @@
+function initToggleShowPassword($password_fields, $password_toggle) {
+
+    /* Private Fields */
+    let timeout_id;
+    let enable_timeout = true;
+
+    const ms_in_sec = 1000;
+    const timeout_ms = 60 * ms_in_sec;
+    const $timeout_message = $("<div />", {
+            class: "small text-muted mb-3",
+            style: "display: none",
+            html: "Passwords will be automatically hidden after " + (timeout_ms / ms_in_sec) + " seconds. <br />" +
```

Comment:
Needs gettext wrappers for localisation

Also a bit nitpicky, but is it better to just use seconds internally and `* 1000` in the `setTimeout`, rather then multiplying then dividing?

And there are a couple of semicolons missing in this file